content/docs/guides/autoscaling-guide.md

```diff
@@ -1,15 +1,52 @@
+subtitle: Learn how to enable Neon's Autoscaling feature to automatically scale compute
+  resources on demand
 enableTableOfContents: true
-updatedOn: '2023-08-09T08:55:53Z'
+updatedOn: '2023-10-17T19:40:28.848Z'
 ---
 
 Neon's Autoscaling feature dynamically adjusts the amount of compute resources allocated to a Neon compute endpoint in response to the current workload, eliminating the need for manual intervention. This guide demonstrates how to enable _Autoscaling_ in your Neon project.
 
-_Autoscaling_ is a [Neon Pro plan](/docs/introduction/pro-plan) feature. Pro plan users can enable Autoscaling for any compute endpoint in a Neon project. _Autoscaling_ is supported with both read-write and read-only compute endpoints. Read-only compute endpoints enable Neon's [Read replica](/docs/introduction/read-replicas) feature.
+_Autoscaling_ is a [Neon Pro plan](/docs/introduction/pro-plan) feature. Pro plan users can enable Autoscaling for a new project, for an existing project, or for an individual compute endpoint. _Autoscaling_ is supported with both read-write and read-only compute endpoints. Read-only compute endpoints enable Neon's [Read replica](/docs/introduction/read-replicas) feature.
 
-You can enable _Autoscaling_ in an existing project by editing a compute endpoint. You can also enable it when you first create a Neon project, which sets the _Autoscaling_ defaults for your project. Both methods are described below.
+## Enable Autoscaling for a new project
+
+Enabling Autoscaling when you create a project allows you to set _Autoscaling_ default settings for all compute endpoints created in your project. You can adjust _Autoscaling_ settings for individual compute endpoints afterward, but setting defaults when creating a project saves you from having to configure the settings for each compute endpoint later on.
+
+To enable Autoscaling at project creation time:
+
+1. Navigate to the [Neon Console](https://console.neon.tech).
+2. If you are creating your very first project, click **Create a project**. Otherwise, click **New Project**.
+3. Specify a name, a Postgres version, and a region.
+4. Under **Compute size**, select the **Autoscaling** option.
+5. Using the slider, specify a minimum and maximum compute size.
+    ![Autoscaling](/docs/guides/autoscaling_project_creation.png)
+
+    <Admonition type="note">
+    You can configure the Auto-suspend setting for your compute endpoint at the same time. The **Suspend compute after a period of inactivity** setting defines the period of inactivity after which a compute endpoint is automatically suspended. This feature is also referred to as "scale-to-zero".
+    </Admonition>
+
+6. Click **Create Project**. Your initial compute endpoint is created with the specified settings. All compute endpoints created in the future when creating a branch or adding a compute endpoint to a branch are created with this setting.
```

Comment:
Consider simplifying to just:

"All future compute endpoints that you create will use this setting"

Reply:
Fixed. TY.